<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>thirdDialog</class>
 <widget class="QDialog" name="thirdDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>831</width>
     <height>621</height>
    </rect>
   </property>
   <property name="title">
    <string>Create Account</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>786</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>       USERNAME                             </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_username"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>      PASSWORD                              </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_password"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>      CONFIRM PASSWORD              </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_confirm"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>      MOBILE NUMBER                      </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_mobilenumber"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>                               GENDER                                                           </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="male">
         <property name="text">
          <string>Male</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="female">
         <property name="text">
          <string>Female</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="other">
         <property name="text">
          <string>Other</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>                              DATE OF BIRTH                                         </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_date">
         <item>
          <property name="text">
           <string>DATE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>01</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>02</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>03</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>04</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>05</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>06</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>07</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>08</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>09</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>17</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>21</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>22</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>23</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>24</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>25</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>26</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>27</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>28</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>29</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>31</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_month">
         <item>
          <property name="text">
           <string>MONTH</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JAN</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FEB</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MAR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>APR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MAY</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JUN</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>JUL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AUG</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SEP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>OCT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NOV</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DEC</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_year">
         <item>
          <property name="text">
           <string>YEAR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2010</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2009</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2008</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2007</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2006</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2005</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2004</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2003</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2002</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2001</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1999</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1998</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1997</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1996</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1995</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1994</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1993</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1992</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1991</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1990</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>      ADDRESS                                   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_address"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>       COUNTRY                                                                               </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_country">
         <item>
          <property name="text">
           <string>SELECT</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>INDIA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CHINA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>USA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RUSSIA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ENGLAND</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>AUSTRALIA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NEW ZEALAND</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>530</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_base">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>600</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>530</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>thirdDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>590</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>683</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
